src/main/java/com/google/api/generator/gapic/composer/utils/ClassNames.java

```diff
@@ -120,4 +123,11 @@ private static String monolithBackwardsCompatibleName(String rawServiceName) {
         ? rawServiceName.replace("IAM", "Iam")
         : rawServiceName;
   }
+
+  // For region tags, the service name should not include `Client` in the name
+  public static String getPureServiceName(String serviceClassName) {
```

Comment:
What I meant was that we can extract the word `Client` to a constant(e.g. `CLIENT_SUFFIX = "Client"`), and make `SERVICE_CLIENT_CLASS_NAME_PATTERN = "%s" + CLIENT_SUFFIX`. So that in case the suffix changes, we don't have to update multiple places.

Reply:
Ah sorry that makes sense. Let me know if the update matches how you were thinking of it -